packages/components/stories/HeaderBar.js

```diff
@@ -98,7 +98,24 @@ const props = {
 	brand: {
 		id: 'header-brand',
 		label: 'Example App Name',
-		onClick: action('onApplicationNameClick'),
+		items: [
```

Comment:
Breaking change spotted

Reply:
Added breaking change log in the description (to report in wiki on merge)